.style.yapf

```diff
@@ -0,0 +1,6 @@
+[style]
+based_on_style: google
+column_limit: 80
```

Comment:
Ew. 100 is better.

(Mostly joking; I prefer 100 for my stuff but it's super trivial and I don't actually care here.)